inst/components/value_box.scss

```diff
@@ -36,6 +36,16 @@
       margin-bottom: 0;
       margin-top: 0;
     }
+    // the title
+    > :first-child { 
+      font-size: $h6-font-size;
+      margin-bottom: $spacer / 4;
+    }
+    // the value
+    > :nth-child(2) { 
+      font-size: $h2-font-size;
+      margin-bottom: $spacer / 2;
+    }
```

Comment:
Am I reading between the lines correctly in guessing that you're using positional selectors here to make something like this easier?

```r
value_box(
  title = uiOutput("vb-title"),
  value = uiOutput("vb-value")
)
```